lib/vanagon/platform/windows.rb

```diff
@@ -37,6 +37,25 @@ def package_name(project)
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_packaging_artifacts(workdir, name, binding)
+        # HOW DO I WORK OUT the Package Type Here - Can I use the binding as per next statement ??
+        # Or should we be splitting out this into a nuget and a windows sub-class - i.e. too many package_type
+        package_type = binding.eval("@platform.package_type")
+        case package_type
```

Comment:
@melissa Hmm trouble is that is generate artefacts is defined for each platform so would I need to fix it in multiple files?

Let's discuss tomorrow.